VoodooPS2Keyboard/VoodooPS2Keyboard.cpp

```diff
@@ -1526,6 +1534,9 @@ bool ApplePS2Keyboard::dispatchKeyboardEventWithPacket(const UInt8* packet)
         case 0x0128:    // alternate that cannot fnkeys toggle (discrete trackpad toggle)
         case 0x0054:    // SysRq (PrntScr when combined with Alt modifier -left or right-)
         {
+            if (!_remapPrntScr)
```

Comment:
I'm not sure if this is necessary

Reply:
This part is introduced in https://github.com/acidanthera/VoodooPS2/commit/19374b1db1b5f1cdd1cc0be02239d85dde301d8b. For 0x128, it makes sense to be re-mapped since that's a dead key. But for 0x0054 sysRq, maybe that could be directly passed to userspace?

Reply:
What is 0x0128? I can't find it, but maybe I'm looking wrong tables.

Reply:
That's a remapped [`e0 27`](https://github.com/acidanthera/VoodooPS2/blob/master/VoodooPS2Keyboard/ApplePS2ToADBMap.h#L911) introduced in https://github.com/acidanthera/VoodooPS2/commit/a123356614d4090569d6dfaf4111b93699b8a8c4